flexmeasures/api/v3_0/sensors.py

```diff
@@ -596,10 +596,10 @@ def post(self, sensor_data: dict):
 
     @route("/<id>", methods=["PATCH"])
     @use_args(partial_sensor_schema)
-    @use_kwargs({"db_sensor": SensorIdField(data_key="id")}, location="path")
-    @permission_required_for_context("update", ctx_arg_name="db_sensor")
+    @use_kwargs({"sensor": SensorIdField(data_key="id")}, location="path")
+    @permission_required_for_context("update", ctx_arg_name="sensor")
     @as_json
-    def patch(self, sensor_data: dict, id: int, db_sensor: Sensor):
+    def patch(self, sensor_data: dict, id: int, sensor: Sensor):
```

Comment:
Should we mention the permissions level in the docstring? (Do we do that for other endpoints?)

Reply:
We should also mention in the docstring that this changing unit, event resolution and knowledge horizon should currently only be done on sensors without data, or if you really know what you are doing.

Reply:
Or could the PATCH endpoint even ask that an optional `force` parameter isvl True if these fields are changed and the sensor has data?

Reply:
As the "proper" solution, we envisioned implementing the logic required to update sensor data corresponding to a change in unit, event resolution or knowledge horizon, with some boolean field to turn on/off such a data update when updating the sensor properties (on by default). But for this release, we just want to make sure users cannot accidentally destroy their data history by e.g. patching a unit without changing the underlying data.

We actually considered three short-term options:
1. only allow patching these properties on empty sensors (patching the name would always be fine)
2. only allow the sensor name to be patched
3. only allow admins to patch sensors

We went with option 3, which we think is the best balance of low implementation effort without diminishing the patch feature too much.

Reply:
I added a comment that only admins are able to use this endpoint

Reply:
I can also do this, it's a quick fix.